Mathlib/Algebra/Group/Finsupp.lean

```diff
@@ -141,6 +141,16 @@ lemma _root_.AddEquiv.finsuppUnique_symm {M : Type*} [AddZeroClass M] (d : M) :
   rw [Finsupp.unique_single (AddEquiv.finsuppUnique.symm d), Finsupp.unique_single_eq_iff]
   simp [AddEquiv.finsuppUnique]
 
+theorem add_comm_of_disjoint {M} {f g : M →₀ N} (h : Disjoint f.support g.support) :
+    f + g = g + f := by
```

Comment:
Could you state this more generally as something like "AddCommute f g \iff \forall x \in f.support, x \in g support -> AddCommute (f x) (g x)`?

Reply:
Sure, I've added the generalization.